Diagnostic/DistroSpecific.py

```diff
@@ -325,8 +325,9 @@ def install_required_packages(self):
     'redhat': RedhatActions, 
     'centos': CentosActions, 
     'oracle': RedhatActions,
-    'SuSE:12': Suse12Actions, 
-    'SuSE': Suse12Actions
+    'SuSE:12': Suse12Actions,
+    'sles:15': Suse12Actions,
+    'opensuse:15':Suse12Actions
```

Comment:
Do we need to be concerned with older versions as well?

Reply:
SuSE 12 is already in the support matrix. (It is the lowest we support)